src/main/java/engineering/swat/watch/Watcher.java

```diff
@@ -146,6 +168,22 @@ public Watcher withExecutor(Executor callbackHandler) {
         return this;
     }
 
+    /**
+     * Optionally configure which regular files/directories in the scope of the
+     * watch an <i>approximation</i> of synthetic events (of kinds
+     * {@link WatchEvent.Kind#CREATED}, {@link WatchEvent.Kind#MODIFIED}, and/or
+     * {@link WatchEvent.Kind#DELETED}) should be issued when an overflow event
+     * happens. If not defined before this watcher is started, the
+     * {@link engineering.swat.watch.OnOverflow#ALL} approach will be used.
+     * @param whichFiles Constant to indicate for which regular
+     * files/directories to approximate
+     * @return This watcher for optional method chaining
+     */
+    public Watcher approximate(OnOverflow whichFiles) {
```

Comment:
The README.md should be updated to clearly describe this functionality (at least the benefit of this library)